src/components/IconButton/IconButton.jsx

```diff
@@ -23,6 +23,7 @@ const IconButton = forwardRef(
       size,
       tooltipContent,
       ariaLabel,
+      showTooltip,
```

Comment:
let's change it to hideTooltip because we prefer always use negative value as default in optional props?

Reply:
Changed 👍